src/Navbar.js

```diff
@@ -3,7 +3,7 @@ import './Navbar.css';
 const Navbar = () => {
```

Comment:
I noticed the logo in the navbar is spilling over the boundaries of the container on both my smaller and larger screen

Reply:
Thanks for fixing that!